config/manufacturer_specific.xml

```diff
@@ -939,7 +939,8 @@
 	</Manufacturer>
 	<Manufacturer id="0021" name="Zykronix">
 	</Manufacturer>
-	<Manufacturer id="0258" name="Neo Coolcam">
+	<Manufacturer id="0258" name="Neo CoolCam">
```

Comment:
Yeah, I noticed the problem after submitting...

2016-08-19 11:21 GMT+02:00 Jean-François Auger <notifications@github.com>:

> In config/manufacturer_specific.xml
> <https://github.com/OpenZWave/open-zwave/pull/970#discussion_r75451190>:
>
> > @@ -939,7 +939,8 @@
> >  	</Manufacturer>
> >  	<Manufacturer id="0021" name="Zykronix">
> >  	</Manufacturer>
> > -	<Manufacturer id="0258" name="Neo Coolcam">
> > +	<Manufacturer id="0258" name="Neo CoolCam">
>
> Please always use latest version before do a PR. I move the Manufacturer
> in right order
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenZWave/open-zwave/pull/970/files/903b0fa18aad2ef41950c2a939fe377c81880cf1..714d466ce83c0f4177724e94970462a2592c0af0#r75451190>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHeDrr-2NXcA7M8HKmuzmFMx0xCbHmr4ks5qhXWIgaJpZM4JnkA3>
> .
>